GoogleTestAdapter/Tests.Common/ResultChecker/TrxResultChecker.cs

```diff
@@ -61,7 +61,8 @@ public void RunTestsAndCheckOutput(string typeName, string arguments, string tes
                 Assert.Inconclusive($"First time this test runs, created golden file - check for correctness! File: {goldenFile}");
             }
             CheckIfFileIsParsable(goldenFile);
-            if (!CompareXmlFiles(goldenFile, transformedResultFile, out string diffFile))
+            string diffFile;
```

Comment:
Yes, that's right. The changes were made to ensure build time compatibility with Visual Studio 2015. As we agreed to drop this you can ignore these changes.